app/src/pages/review/[resourceType]/[id].tsx

```diff
@@ -0,0 +1,68 @@
+import { Prism } from '@mantine/prism';
+import { Center, Divider, Grid, Paper, Space, Tabs, Text } from '@mantine/core';
+import { ArtifactResourceType } from '@/util/types/fhir';
+import { useRouter } from 'next/router';
+import { trpc } from '@/util/trpc';
+
+/**
+ * Component which renders a page that displays the JSON data of a resource. Also will eventually
+ *  provide the user with the ability to make review comments and visualize previous review comments.
+ */
+export default function CommentPage() {
+  const router = useRouter();
+  const { resourceType: resourceType, id: resourceID } = router.query;
+  const { authoring } = router.query;
+  let resource;
+
+  if (authoring == 'true') {
+    resource = trpc.draft.getDraftById.useQuery({
+      id: resourceID as string,
+      resourceType: resourceType as ArtifactResourceType
+    });
+  } else {
+    resource = trpc.service.getArtifactById.useQuery({
+      id: resourceID as string,
+      resourceType: resourceType as ArtifactResourceType
+    });
+  }
+
+  return (
+    <div>
+      <Center>
+        <Text c="gray" fz="xl">
+          {authoring === 'true'
+            ? `Reviewing Draft ${resourceType}/${resourceID}`
+            : `Reviewing ${resourceType}/${resourceID}`}
+        </Text>
+      </Center>
+      <Divider my="md" style={{ marginTop: '14px' }} />
```

Comment:
To make better use of the `Divider` props, we should use `mt` instead of `style`. I think this should achieve the same styling:
```suggestion
      <Divider my="md" mt={14} />
```